### PR TITLE
Add a CHANGELOG entry about deposit tree snapshot bundles

### PR DESCRIPTION
**Pull request by zilm13**
## PR Description
The goal is to get some beta-testing before moving out of `--X` or enabling by default

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,5 +18,6 @@ For information on changes in released versions of Teku, see the [releases page]
 - The commandline option `--network` of the `validator-client` subcommand has been **un**deprecated for values other than `auto`.
 - Send validator registrations to the Beacon node when the Validator client has reconnected to the event stream
 - Added optional query parameters `require_prepared_proposers` and `require_validator_registrations` to the `teku/v1/admin/readiness` endpoint
+- Added experimental feature `--Xdeposit-snapshot-enabled` to use bundled deposit contract tree snapshot and persist it after finalization to decrease EL pressure and speed up restart peer availability
```
Comment (tbenr) [inline CHANGELOG.md]: what do you mean by `speed up restart peer availability` ?

Comment (zilm13) [reply CHANGELOG.md]: I mean peer will be able to propose sooner. It's a rare case, yeah, we could omit it. We will not speed up attestation availability

Comment (tbenr) [reply CHANGELOG.md]: so simply "speed up node startup" might be enough

**Review by tbenr — APPROVED** [review]
LGTM

**Pull request merged**